package.json

```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/angular/angular-cli.git"
   },
   "engines": {
-    "node": ">= 4.1.0"
+    "node": ">= 5.0.0"
```

Comment:
I thought you were asking about if we still tested on node 4. It's fine to keep node 4.1, unless there's a technical problem with this PR.